src/test/ui/consts/convert.rs

```diff
@@ -0,0 +1,20 @@
+// run-pass
+
+#![feature(const_trait_impl)]
+#![feature(const_identity_convert)]
+
+fn main() {
```

Comment:
Should the tests be moved to `library/core/tests` now since the try-operator itself was constified in #87237?

Reply:
we've usually had these tests in ui tests, but I think you are right, we should put more const tests for libcore into the libcore tests.

Reply:
Would `library/core/tests/convert.rs` work? Or do you have any other suggestions?

Reply:
yea, that fits